### ISSUE TITLE
GetSamples "logging" element not taken into account

### ISSUE THREAD

**Issue by qlemaire** [DFIR-ORC/dfir-orc#18]
Hello,

While testing `GetSamples` command, I tried to display **verbose** information via XML configuration file (cf https://dfir-orc.github.io/GetSamples.html#output):
```xml
<getsamples nolimits="">
        <logging verbose="" debug="" />
</getsamples>
```

Here is the commandline:
```dos
C:\> DFIR-Orc_x64.exe GetSamples /config=test.xml
```

The `nolimits` attribute seems to be taken into account (otherwise, the command wouldn't execute), but not the "logging" element:
![image](https://user-images.githubusercontent.com/29045289/77626527-ce224380-6f45-11ea-940d-173c0b670159.png)

The "equivalent" commandline is verbose, as expected:
```dos
C:\> DFIR-Orc_x64.exe GetSamples /verbose /debug /nolimits
```

![image](https://user-images.githubusercontent.com/29045289/77626983-70dac200-6f46-11ea-8ad4-3329efad3a3c.png)

Am I missing anything?

Thanks

**Comment by jgautier-anssi** [comment]
Hi Quentin,

We have reproduced this issue internally.
We'll have a fix for this.

Thank you for the report.
Jean

**Comment by fabienfl-orc** [comment]
Affects multiple commands. Should be fix with 10.1.0-rc4

**Comment by fabienfl-orc** [comment]
Configuration should be possible with upcoming version. This version should correctly forward log options to GetThis process with such GetSamples Configuration:
```xml
    <log>
        <file level="debug" backtrace="error">
            <output disposition="truncate">GetSamples.log</output>
        </file>
    </log>
```
